client/src/index.jsx

```diff
@@ -2,77 +2,139 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import styled from 'styled-components';
 
-class App extends React.Component {
+export class App extends React.Component {
   constructor(props){
     super(props);
     this.state = {
-      items: []
+      firstPage: true,
+      item1: [],
+      item2: []
     };
     this.veryFetching = this.veryFetching.bind(this);
+    this.switchUp = this.switchUp.bind(this);
   }
 
   componentWillMount(){
     let prodId = window.location.search.split("=")[1];
-    if (prodId){
+    if (prodId == 1){
       console.log("prodId: ",prodId);
       fetch(`/suggested?prod_id=${prodId}`)
       .then(r => r.json())
       .then(d => {
+        let x = d.slice(0,5);
```

Comment:
No need to assign d.slice(0,5) in a variable if u r not reusing it. You can directly assign it to items1 in setState.